src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -634,12 +634,27 @@ def merge(self,
     # TODO(flowerhack): Integrate some notion of a merge timeout.
     self._push_corpora_from_host_to_target(corpus_directories)
 
+    target_tmp_dir = None
+    if tmp_dir:
+      # Tmp dir may have artifacts, e.g. merge control file for two step merge.
+      target_tmp_dir = self._corpus_target_subdir(os.path.basename(tmp_dir))
+      self.fuzzer.device.rm(target_tmp_dir, recursive=True)
+      self.fuzzer.device.store(tmp_dir, target_tmp_dir)
+      additional_args = copy.copy(additional_args)
```

Comment:
nit: How about `additional_args = [arg if arg != tmp_dir else target_tmp_dir for arg in additional args]`
Maybe not since the rest of the file copies it (probably a flaw in the API, `additional_args` should probably be safe to mutate).

Reply:
the catch here is that arguments might contain more that paths, e.g. `-merge-control-file=<path>